modules/adtelligentBidAdapter.js

```diff
@@ -24,6 +24,7 @@ const HOST_GETTERS = {
   pgam: () => 'ghb.pgamssp.com',
   ocm: () => 'ghb.cenarius.orangeclickmedia.com',
   vidcrunchllc: () => 'ghb.platform.vidcrunch.com',
+  '9dotsmedia': () => 'ghb.platform.audiodots.com'
```

Comment:
Why is this the only one in quotes

Reply:
do you suggest wrapping all keys in quotes ?